docs/_includes/generated/yarn_config_configuration.html

```diff
@@ -14,6 +14,20 @@
             <td>String</td>
             <td>If configured, Flink will add this key to the resource profile of container request to Yarn. The value will be set to the value of external-resource.&lt;resource_name&gt;.amount.</td>
         </tr>
+        <tr>
+            <td><h5>flink.hadoop.&lt;hadoop_key&gt;</h5></td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>String</td>
+            <td>A general option to probe Hadoop configuration through prefix `flink.hadoop.`. Flink will remove the prefix to get hadoop_key (from <a href="https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-common/core-default.xml">core-default.xml</a> and <a href="https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-hdfs/hdfs-default.xml">hdfs-default.xml</a>) then set the hadoop_key and value to Hadoop configuration. For example, flink.hadoop.dfs.replication=5 in Flink configuration and convert to dfs.replication=5 in Hadoop configuration.
+            </td>
+        </tr>
+        <tr>
+            <td><h5>flink.yarn.&lt;key&gt;</h5></td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>String</td>
+            <td>A general option to probe Yarn configuration through prefix `flink.yarn.`. Flink will remove the prefix `flink.` to get yarn_key (from <a href="https://hadoop.apache.org/docs/stable/hadoop-yarn/hadoop-yarn-common/yarn-default.xml">yarn-default.xml</a>) then set the yarn_key and value to Yarn configuration. For example, flink.yarn.resourcemanager.container.liveness-monitor.interval-ms=300000 in Flink configuration and convert to yarn.resourcemanager.container.liveness-monitor.interval-ms=300000 in Yarn configuration.
+            </td>
+        </tr>
```

Comment:
This file is generated. I think we can add a `ConfigOption` with the key `flink.yarn.<key>` and `flink.hadoop.<key>` and the corresponding descriptions to `YarnConfigOptions`.

Reply:
Thanks for your guidance, I understand its generation mechanism and I will add it.

Reply:
@tillrohrmann I have added two options `flink.yarn.<key>` and `flink.hadoop.<key>` in `YarnConfigOptions` and auto generated related description in HTML document.
Then please take a look at this modification and evaluate the completion of this pr, thx!